minsar/utils/download_asfserial_rsmas.py

```diff
@@ -57,8 +57,29 @@ def main(iargs=None):
         os.remove(os.path.expanduser('~') + '/.bulk_download_cookiejar.txt')
     except OSError:
         pass
-
-    generate_files_csv(project_slc_dir, inps.custom_template_file)
+    
+    dataset_template = Template(inps.custom_template_file)
+    dataset_template.options.update(PathFind.correct_for_ssara_date_format(dataset_template.options))
+    subprocess.Popen("rm new_files.csv", shell=True).wait()
+    standardTuple = (inps, dataset_template)
+    if inps.seasonalStartDate is not None and inps.seasonalEndDate is not None:
+        ogStartYearInt = int(dataset_template.options['ssaraopt.startDate'][:4])
+        if int(inps.seasonalStartDate) > int(inps.seasonalEndDate):
+            y = 1
+        else:
+            y = 0
+        YearRange = int(dataset_template.options['ssaraopt.endDate'][:4]) - ogStartYearInt + 1
```

Comment:
Variables should follow underscore caseing. `year_range` not `YearRange`

Reply:
Okay